content/d3/concepts/styling/terms/style/style.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '.style()'
+Description: 'Used to get or set inline properties of selected elememts.'
+Subjects:
+  - 'Data Visualization'
+  - 'Computer Science'
+Tags:
+  - 'D3'
+  - 'Style'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/computer-science'
+---
+
+the **`.style()`** method is used to set or get the inline style properties of selected elements. This method is useful for applying CSS styles to elements.
+
+## Syntax
+
+```pseudo
+selection.style("property", "value");
+```
+
+`.style()` funtion takes two arguements, first is the style name and second is the value to set to it.
+
+## Example
+
+```js
+d3.select('text').style('color', 'red');
+```
+
+In the given example above, the `color` property of `text` is set to `red`.
```

Comment:
can we attach a media below this show what the snippet does?

Reply:
@vrun1208 that's what I was asking if it's possible for you to attack a output sample using either an image or gif. If not then that's okay too.

Reply:
@dakshdeepHERE , I think there's no need for it as sufficient information is provided

Reply:
Alright